openedx/core/djangoapps/video_config/models.py

```diff
@@ -89,30 +111,12 @@ def __unicode__(self):
         )
 
 
-class CourseHLSPlaybackEnabledFlag(ConfigurationModel):
-    """
-    Enables HLS Playback for a specific course. Global feature must be
-    enabled for this to take effect.
-    """
-    KEY_FIELDS = ('course_id',)
-
-    course_id = CourseKeyField(max_length=255, db_index=True)
-
-    def __unicode__(self):
-        not_en = "Not "
-        if self.enabled:
-            not_en = ""
-
-        return u"Course '{course_key}': HLS Playback {not_enabled}Enabled".format(
-            course_key=unicode(self.course_id),
-            not_enabled=not_en
-        )
-
-
 class CourseVideoTranscriptEnabledFlag(ConfigurationModel):
     """
     Enables Video Transcript for a specific course. Global feature must be
     enabled for this to take effect.
+    When this feature is enabled, 3rd party transcript integration functionality would be available for the
```

Comment:
We intend to use this flag in all the phases. So, its not just for integration -- more suitable one would be:
'''
When this feature is enabled,3rd party transcript integration functionality would be available and S3 video transcript would be served (currently as a fallback)
'''